src/ngraph/runtime/backend.cpp

```diff
@@ -85,3 +85,8 @@ bool runtime::Backend::is_supported_property(const Property prop) const
 void runtime::Backend::remove_compiled_function(std::shared_ptr<Executable> exec)
 {
 }
+
+std::shared_ptr<runtime::Executable> runtime::Backend::load(istream& input_stream)
+{
```

Comment:
Add some error message or better - make it a pure virtual. This will help get errors early as opposed to silently failing the function.

Reply:
Same as above